syntax = "proto3";

package PbLapCounter;

message Lap
{
	uint64 timestamp = 1;
	int32 number = 2;
	double time = 3;
	repeated double sectorTimes = 4;
}

message Race
{
	Lap bestLap = 1;
	repeated Lap laps = 2;
}